pkg/injector/injector_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/dapr/dapr/pkg/injector/namespacednamematcher"
```

Comment:
Please put this in the same group as the other `github.com/dapr*` dependencies. The linter will complain